charts/spark-rss/README.md

```diff
@@ -59,7 +59,7 @@ Given below is an example spark-submit command that uses resource staging server
       --deploy-mode cluster \
       --class org.apache.spark.examples.SparkPi \
       --master k8s://<k8s-apiserver-host>:<k8s-apiserver-port> \
-      --kubernetes-namespace default \
+      --conf spark.kubernetes.shuffle.namespace=spark
```

Comment:
Why '...shuffle.namespace' for RSS example, instead of 'spark.kubernetes.namespace' (if not '--kubernetes-namespace')?

Reply:
Copy-paste error. Should be spark.kubernetes.namespace